### ISSUE TITLE
Can't convert mobilenet v2 quantized model from tensorflow to Onnx/Pytorch

### ISSUE THREAD

**Issue by dibya001** [microsoft/MMdnn#696]
Platform (like ubuntu 16.04/win10): 16.04

Python version: 2.7

Source framework with version (like Tensorflow 1.4.1 with GPU): 1.11

Destination framework with version (like CNTK 2.3 with GPU): Onnx

Pre-trained model path (webpath or webdisk path):
model link:- http://download.tensorflow.org/models/tflite_11_05_08/mobilenet_v2_1.0_224_quant.tgz
Running scripts:
Command:- mmconvert -sf tensorflow -iw mobilenet_v2_1.0_224_quant_frozen.pb --inNodeName input --inputShape 224,224,3 --dstNodeName output -df pytorch -om generated

Error Message:-

* Tensorflow has not supported operator [FakeQuantWithMinMaxVars] with name [MobilenetV2/Logits/Conv2d_1c_1x1/weights_quant/FakeQuantWithMinMaxVars] *

venvs/testenv/local/lib/python2.7/site-packages/mmdnn/conversion/tensorflow/tensorflow_frozenparser.py", line 448, in rename_UNKNOWN
    assert False
AssertionError

**Comment by rainLiuplus** [comment]
Hi @dibya001. Thanks for your feedback. It is a new op that we will add in the future.
